src/lib.rs

```diff
@@ -403,12 +408,10 @@ impl Mastodon {
         (delete) delete_status: "statuses/{}" => Empty,
     }
 
-    pub fn update_credentials(&self, changes: CredientialsBuilder)
-        -> Result<Account>
-    {
-
+    pub fn update_credentials(&self, changes: CredientialsBuilder) -> Result<Account> {
         let url = self.route("/api/v1/accounts/update_credentials");
-        let response = self.client.patch(&url)
+        let response = self.client
+            .patch(&url)
```

Comment:
Revert this change please, semantically I prefer to start newline chaining functions with the first function with the object.